packages/office-ui-fabric-react/src/utilities/grid/GridCell.Props.ts

```diff
@@ -0,0 +1,62 @@
+export interface IGridCellProps {
+
+  /**
+   * The option that will be made available to the user
+   */
+  item: any;
+
+  /**
+   * Arbitrary unique string associated with this option
+   */
+  id: string;
+
+  /**
+   * Optional, if the this option should be diabled
+   */
+  disabled?: boolean;
+
+  /**
+   * Optional, the currently selectedIndex in the set of options
+   */
+  selectedIndex?: number;
```

Comment:
can we make this selected? : boolean and not pass in the selected index? The parent should determine if this is true.